packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/CameraRegionUtils.java

```diff
@@ -0,0 +1,138 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera;
+
+import android.annotation.TargetApi;
+import android.hardware.camera2.CaptureRequest;
+import android.hardware.camera2.params.MeteringRectangle;
+import android.os.Build;
+import android.util.Size;
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import java.util.Arrays;
+
+/**
+ * Utility class offering functions to calculate values regarding the camera boundaries.
+ *
+ * <p>The functions are used to calculate focus and exposure settings.
+ */
+public final class CameraRegionUtils {
+
+  /**
+   * Obtains the boundaries for the currently active camera, that can be used for calculating
+   * MeteringRectangle instances required for setting focus or exposure settings.
+   *
+   * @param cameraProperties - Collection of the characteristics for the current camera device.
+   * @param requestBuilder - The request builder for the current capture request.
+   * @return The boundaries for the current camera device.
+   */
+  public static Size getCameraBoundaries(
+      @NonNull CameraProperties cameraProperties, @NonNull CaptureRequest.Builder requestBuilder) {
+    // No distortion correction support
```

Comment:
This comment should be inside the `else` block; right now it's actively confusing since it looks like it's documenting the `if` condition.